wolfcrypt/src/rsa.c

```diff
@@ -3373,7 +3373,23 @@ static int RsaPublicEncryptEx(const byte* in, word32 inLen, byte* out,
                                   pad_value, pad_type, hash, mgf, label,
                                   labelSz, sz);
         }
-    #endif /* WOLFSSL_CRYPTOCELL */
+    #elif defined(WOLFSSL_RENESAS_SCEPROTECT_CRYPTONLY)
+           /* SCE needs warpped key which is passed via
+            * user ctx object of crypt-call back.
+            */
+           #ifdef WOLF_CRYPTO_CB
+            if (key->devId != INVALID_DEVID) {
```

Comment:
Our indentation level should be 4 spaces.  It looks like you only have one space indent for the block inside `#ifdef WOLF_CRYPTO_CB`.

Reply:
Thanks! Fixed.